controllers/config_helper.go

```diff
@@ -0,0 +1,192 @@
+package controllers
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/go-logr/logr"
+	"github.com/open-policy-agent/gatekeeper/v3/apis/config/v1alpha1"
+	"github.com/pkg/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"k8s.io/client-go/dynamic"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/metrics/server"
+)
+
+var errConfigCrdNotReady = errors.New("Config CRD is not ready")
+
+// Default config data
+var configDefaultResource = []byte(`
+apiVersion: config.gatekeeper.sh/v1alpha1
+kind: Config
+metadata:
+  name: config
+spec:
+  match:
+  - excludedNamespaces: ["kube-*", "multicluster-engine", "hypershift", "hive", "rhacs-operator", "open-cluster-*"] 
+    processes: ["audit", "webhook", "sync"]
+`)
+
+func createConfigFromYaml(ctx context.Context, c client.Client, namespace string) error {
+	config := &v1alpha1.Config{}
+
+	err := yaml.Unmarshal(configDefaultResource, config)
```

Comment:
Why not just set this directly with `config`? Seems much cleaner than unmarshaling the YAML. You'd also get feedback of syntax errors at compile time instead of runtime.

Reply:
oh I thought unmarshaling the YAML way is more clear when someone want to modify excludedNamespaces

Reply:
I would personally have a function of `getDefaultExcludedNamespaces` that returns the Go type that you put in `Config`.